test/eventlistener_test.go

```diff
@@ -383,18 +385,6 @@ func TestEventListenerCreate(t *testing.T) {
 		t.Fatalf("Failed to delete EventListener Service: %s", err)
 	}
 	t.Log("EventListener's Service was deleted")
-
-	// Cleanup cluster-scoped resources
-	t.Logf("Deleting cluster-scoped resources")
-	if err := c.KubeClient.RbacV1().ClusterRoles().Delete("my-role", &metav1.DeleteOptions{}); err != nil {
-		t.Errorf("Failed to delete clusterrole my-role: %s", err)
-	}
-	if err := c.KubeClient.RbacV1().ClusterRoleBindings().Delete("my-rolebinding", &metav1.DeleteOptions{}); err != nil {
-		t.Errorf("Failed to delete clusterrolebinding my-rolebinding: %s", err)
-	}
-	if err := c.TriggersClient.TektonV1alpha1().ClusterTriggerBindings().Delete("my-clustertriggerbinding", &metav1.DeleteOptions{}); err != nil {
-		t.Errorf("Failed to delete clustertriggerbinding my-clustertriggerbinding: %s", err)
-	}
```

Comment:
I think that these test cleanup changes belong in a separate PR. There're so many changed files to dig through in this PR, I don't want these changes to get lost in all of it.

Reply:
fair point though I needed it to test it out locally. Opened #503